src/pages/docs/app-manifest.mdx

```diff
@@ -264,35 +264,41 @@ Available naming related string transforming functions:
 
   </Field>
 
-  <Field
-    meta={{
-      path: "components.<component-name>.files[*].sourcePath",
-    }}
-  >
-    ***Required source path*** of the file to be added to the _Initial File System_. The path can be a
-    _file path_ relative to the manifest document or an _URL_.
-  </Field>
-
-  <Field
-    meta={{
-      path: "components.<component-name>.files[*].targetPath",
-    }}
-  >
-    ***Required target path*** of the file in the _Initial File System_. The path must be an absolute
-    path.
-  </Field>
-
-  <Field
-    meta={{
-      path: "components.<component-name>.files[*].permissions",
-    }}
-  >
-    ***Optional string enum*** which controls file permissions. Accepted values:
-    <EnumValues>
-      <EnumValue meta={{value: "read-only", isDefault: true}}></EnumValue>
-      <EnumValue meta={{value: "read-write"}}></EnumValue>
-    </EnumValues>
-  </Field>
+{" "}
```

Comment:
Identation is a bit inconsistent here, rest of the field references is indented with 2 spaces